cypress/integration/data-import/mrc-import-file-for-create-items.spec.js

```diff
@@ -88,10 +88,13 @@ describe('ui-data-import: MARC file import with creating of the new instance, ho
     DataImport.uploadFile('oneMarcBib.mrc', fileName);
     JobProfiles.searchJobProfileForImport(specialJobProfile.profileName);
     JobProfiles.runImportFile(fileName);
-    Logs.checkImportFile(specialJobProfile.profileName);
     Logs.checkStatusOfJobProfile();
+    Logs.checkImportFile(specialJobProfile.profileName);
     Logs.openFileDetails(fileName);
-    FileDetails.checkCreatedItems();
+    FileDetails.checkStatusInColumn(FileDetails.status.created, FileDetails.columnName.srsMarc);
```

Comment:
you can try to run foreach for FileDetails.columnName in this case

Reply:
Done